### ISSUE TITLE
ERR_MODULE_NOT_FOUND with ESM imports

### ISSUE THREAD

**Issue by jakeklassen** [plexinc/papr#121]
Hello, I'm really looking forward to testing this library but I ran into an import issue using TypeScript + esbuild + ESM.

```
Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/home/user/code/project/node_modules/papr/esm/model' imported from /home/user/code/project/node_modules/papr/esm/index.js
    at new NodeError (node:internal/errors:371:5)
    at finalizeResolution (node:internal/modules/esm/resolve:416:11)
    at moduleResolve (node:internal/modules/esm/resolve:932:10)
    at defaultResolve (node:internal/modules/esm/resolve:1044:11)
    at ESMLoader.resolve (node:internal/modules/esm/loader:422:30)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:222:40)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:76:40)
    at link (node:internal/modules/esm/module_job:75:36) {
  code: 'ERR_MODULE_NOT_FOUND'
  ```

Curious if anyone else has ran into this. I'm relatively new to using esbuild so could be some simple mistakes on my end. Builds with both tsc and esbuild succeed, this is happening when I run the build output.

My file for testing:

```typescript
import { MongoClient } from 'mongodb';
import Papr from 'papr';

const papr = new Papr();

const connection = await MongoClient.connect(
  'mongodb://localhost:27017',
);
papr.initialize(connection.db('tantor'));

await papr.updateSchemas();
```

**Comment by benheller** [comment]
I just ran into this too -- the issue here seems to be that the imports in `paper/esm/index.js` don't include the `.js` file extension, such that when running in vanilla node the paths to `model` and other package-relative imports are incorrect. 

Add `--experimental-specifier-resolution=node` to the command and that'll do it.

**Comment by avaly** [comment]
Fix available in v2.1.1
